lib/rubocop/plugin/plugin_config_normalizer.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Plugin
+    # Normalize and structure configuration settings for plugins.
+    # Ensures compatibility and default values for plugin-related configurations.
+    # @api private
+    class PluginConfigNormalizer
```

Comment:
I'm not sure this is needed as a separate class as well, given that it just has a single class method.

Reply:
This has been inlined into `RuboCop::Plugin::Loader`.